src/commands/create/contract.ts

```diff
@@ -160,6 +151,8 @@ export default class Contract extends Command {
     let signature: Signature
     let needToSign = false
 
+    const METADATA_RENDERER_ADDRESS = '0x03714aCb8E3325E43d967A2549541295a672d999'
```

Comment:
Use `ENVIRONMENT` as a key for this map so that it loads proper address.

Reply:
done